.editorconfig

```
@@ -0,0 +1,22 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.json]
+[*.{json,yaml,yml}]
```

I don't think we need both.

```suggestion
[*.{json,yaml,yml}]
```